app/api/webhook/github/route.ts

```diff
@@ -344,7 +378,8 @@ ${isComment ? `Comment by @${comment.user.login}: ${comment?.body}.` : ''}
 			octokit,
 			prisma,
 			customerId,
-			repoName: name
+			repoName: `${owner}/${name}`,
```

Comment:
Only downstream effect was in `prisma.project.update({ where: { ... name: repoName } ... })` in the `updateInstructions` tool